docs/setup/deployment.md

```diff
@@ -88,7 +97,7 @@ requirements are Python, a web server, an application server, and a database.
 - Python 3.7+, pip, pipenv
 - Web server ([nginx](http://nginx.org/), [Apache](http://httpd.apache.org/), etc.)
 - Application server ([uwsgi](http://projects.unbit.it/uwsgi), [gunicorn](http://gunicorn.org/), etc.)
-- Database ([sqlite](https://sqlite.org/), [Postgres](https://www.postgresql.org/), [MySQL](https://www.mysql.com/), etc.)
+- Database (See [Databases requirements](#databases)).
```

Comment:
Instead of re-creating the list, let's just say here: See [Django's databases documentation](https://docs.djangoproject.com/en/4.2/ref/databases/)

Reply:
Okay, changes done.